iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_rgb_camera.ini

```diff
@@ -0,0 +1,7 @@
+[include "gazebo_icub_robotname.ini"]
+
+device grabberDual
```

Comment:
Not related to this PR specifically, but I would avoid using devices that are deprecated, could you please use `frameGrabber_nws_yarp` instead?
Thanks!

Reply:
Thanks for the heads up about the cameras deprecated devices. Done in https://github.com/mebbaid/icub-models/commit/91a5d1c4a362faafeb5498f4effb400d6f3b30d3 and tested.